src/main/java/seedu/address/logic/filters/CarFilter.java

```diff
@@ -21,17 +21,12 @@ public CarFilter(String filterString) {
     /**
      * Parses filter command for c/, cp/ prefixes.
      *
-     * @param car
+     * @param car the car object to parse
      * @return
      */
     public static Car parseCar(String car) {
-        String trimmedCar = car;
-        String[] carDetails = trimmedCar.split("\\+");
+        String[] carDetails = car.split("\\+");
 
-
-        if (carDetails.length != 2) {
-            //throw new ParseException(Car.MESSAGE_CONSTRAINTS);
-        }
```

Comment:
Should this be deleted, or should we throw an exception upon `carDetails.length > 2`